src/main/java/com/adobe/epubcheck/opf/OPFChecker30.java

```diff
@@ -395,6 +397,26 @@ private void checkPagination()
     }
   }
 
+  private void checkLanguage() {
+    Set<Metadata> dcLanguageMetas = ((OPFHandler30) opfHandler).getMetadata()
+            .getPrimary(DCMESVocab.VOCAB.get(DCMESVocab.PROPERTIES.LANGUAGE));
+    if (!dcLanguageMetas.isEmpty())
+    {
+      for(Metadata metadata : dcLanguageMetas)
+      {
+        Locale l = Locale.forLanguageTag(metadata.getValue());
+        if (l == null || l.getLanguage().length() > 3) {
+          report.message(MessageId.OPF_086, EPUBLocation.create(path), metadata.getValue());
```

Comment:
OK. 
A more thorough validation would be to use [IANA’s language tag registry](https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry), but this is probably overkill?